src/components/RevokeBlock.tsx

```diff
@@ -34,6 +15,7 @@ export const RevokeBlock: FunctionComponent<DocumentStoreAddressProp> = ({ docum
   const [logs, setLogs] = useState("");
```

Comment:
it's a bit weird to be plural while it's a single value. What's the reason?

Reply:
will change it to singular

Reply:
issue fixed at #26